protocol/communities/manager.go

```diff
@@ -4129,3 +4117,51 @@ func (m *Manager) fixupChannelMembers() error {
 
 	return nil
 }
+
+func (m *Manager) ReevaluatePrivelegedMember(community *Community, tokenPermissions []*protobuf.CommunityTokenPermission,
+	accountsAndChainIDs []*AccountChainIDsCombination, memberPubKey *ecdsa.PublicKey,
+	privilegedRole protobuf.CommunityMember_Roles, hasPrivilegedRole bool) (bool, error) {
```

Comment:
These two arguments, `privilegedRole` and `hasPrivilegedRole`, next to each other are a bit confusing.
```suggestion
	privilegedRole protobuf.CommunityMember_Roles, alreadyHasPrivilegedRole bool) (bool, error) {
```